src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml.cs

```diff
@@ -132,7 +155,10 @@ public async Task<IActionResult> OnPostUploadAsync()
             {
                 UserName = User.GetGitHubLogin(),
                 FilterType = filterTypes.Cast<ReviewType>().ToList(),
-                Language = languages
+                Search = search,
```

Comment:
I assume UI shows this search keywords in search text box as well. How does this work when user opens a new tab? Will it also show previous search results?

Reply:
I think search seems a little overkill to put it in user preference.

Reply:
Ok, I will remove search.